app/views/facility_accounts_reconciliation/_table.html.haml

```diff
@@ -3,9 +3,9 @@
     %tr
       %th
       %th= Statement.human_attribute_name(:invoice_number)
-      %th= t(".th.order")
+      %th= Order.model_name.human
       %th= Statement.human_attribute_name(:total_cost)
-      %th= t(".th.reconcile_note")
+      %th= Statement.human_attribute_name(:reconcile_note)
```

Comment:
Should we `titlecase` this? We're not really consistent on this; as an example just above we've got `Statement.human_attribute_name(:invoice_number)` not titlecased.